psi4/src/psi4/libmints/eri.cc

```diff
@@ -462,3 +462,103 @@ void Libint2ErfComplementERI::libint2_wrapper2(const libint2::Shell &sh1, const
 }
 
 Libint2ErfComplementERI::~Libint2ErfComplementERI(){};
+
+//// Libint2 implementation
+Libint2YukawaERI::Libint2YukawaERI(double zeta, const IntegralFactory *integral, double screening_threshold, int deriv, 
+                                    bool use_shell_pairs, bool needs_exchange)
+    : Libint2TwoElectronInt(integral, deriv, screening_threshold, use_shell_pairs, needs_exchange) {
+    timer_on("Libint2YukawaERI::Libint2YukawaERI");
+    int max_am =
+        std::max(std::max(basis1()->max_am(), basis2()->max_am()), std::max(basis3()->max_am(), basis4()->max_am()));
+    int max_nprim = std::max(std::max(basis1()->max_nprimitive(), basis2()->max_nprimitive()),
+                             std::max(basis3()->max_nprimitive(), basis4()->max_nprimitive()));
+    const auto max_precision = 0.;  // equivalent in accuracy and timings is `std::numeric_limits<double>::epsilon() * 1e-30;`
+
+    bool dummy1 = basis1()->l2_shell(0) == libint2::Shell::unit();
+    bool dummy2 = basis2()->l2_shell(0) == libint2::Shell::unit();
+    bool dummy3 = basis3()->l2_shell(0) == libint2::Shell::unit();
+    bool dummy4 = basis4()->l2_shell(0) == libint2::Shell::unit();
+
+    if (!dummy1 && !dummy2 && !dummy3 && !dummy4) {
+        braket_ = libint2::BraKet::xx_xx;
+    } else if (!dummy1 && dummy2 && !dummy3 && !dummy4) {
+        braket_ = libint2::BraKet::xs_xx;
+    } else if (!dummy1 && !dummy2 && !dummy3 && dummy4) {
+        braket_ = libint2::BraKet::xx_xs;
+    } else if (!dummy1 && dummy2 && !dummy3 && dummy4) {
+        braket_ = libint2::BraKet::xs_xs;
+    } else {
+        throw PSIEXCEPTION("Bad BraKet type in Libint2TwoElectronInt");
```

Comment:
```suggestion
        throw PSIEXCEPTION("Bad BraKet type in Libint2YukawaERI");
```
questionable, as preceding two-electron ints types do this, too